src/workspace/workspaceMonitor.ts

```diff
@@ -131,7 +133,10 @@ export class WorkspaceMonitor implements vscode.Disposable {
 		this.maybeNotifyOutdated(workspace);
 		this.maybeNotifyAutostop(workspace);
 		this.maybeNotifyDeletion(workspace);
-		this.maybeNotifyNotRunning(workspace);
+		if (this.isReady) {
+			// This instance might be created before the workspace is running
+			this.maybeNotifyNotRunning(workspace);
+		}
```

Comment:
This notification can be a little bit weird, if I stop my workspace while trying to connect for example it says "reload to reconnect" but actually in the background we are handling it and reconnecting automatically.  Maybe we only show it after fully connecting rather than when ready?  Not sure.